components/automate-ui/src/app/modules/policy/list/policy-list.component.ts

```diff
@@ -60,9 +61,11 @@ export class PolicyListComponent implements OnInit, OnDestroy {
     this.deleteModalVisible = false;
   }
 
-  public startPolicyDelete(policy: Policy): void {
-    this.deleteModalVisible = true;
-    this.policyToDelete = policy;
+  public startPolicyDelete($event: ChefKeyboardEvent, policy: Policy): void {
+    if ($event.isUserInput) {
```

Comment:
This is effect of using Material UI's `mat-select` component.  When a selection is made, it fires off two events instead of just one.  One of the events is the item being selected, and the other event is something being DEselected.

Unfortunately, without this gate being added, we'd fire off two events.  The workaround is to check for the `isUserInput` property being true, if it is - that's the selected event we want.